hangman.py

```diff
@@ -0,0 +1,330 @@
+        list_guessed_word[-1] = '_'
+    return ''.join(list_guessed_word)
+
+
+def get_available_letters(letters_guessed):
+    '''
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    '''
+    list_of_avaible_letters = []
+    for letters in string.ascii_lowercase:
+        if letters not in letters_guessed:
+            list_of_avaible_letters .append(letters)
+    return ''.join(list_of_avaible_letters)
+
+
+def output_information(guesses_left, letters_guessed):
+    '''
+    Main information for the player
+    '''
+    print("You have", guesses_left, "guesses left")
+    get_available_letters(letters_guessed)
+    print("Available letters:", get_available_letters(letters_guessed))
+    guessed_letter = input("Please guess a letter: ")
+    return guessed_letter
+
+
+def not_correct_input(warnings_left, guesses_left, secret_word, letters_guessed):
+    '''
+    System of losing warnings or guesses when letter wasn't input
+    '''
+    if warnings_left <= 0:
+        guesses_left -= 1
+        print("Oops! That is not a valid letter. You have no warnings left so you lose one guess:",
+              get_guessed_word(secret_word, letters_guessed))
+        return guesses_left
+    else:
+        warnings_left -= 1
+        print("Oops! That is not a valid letter. You have", warnings_left, "warnings left:",
+              get_guessed_word(secret_word, letters_guessed))
+        return warnings_left
+
+
+def guessed_before(warnings_left, guesses_left, letters_guessed):
+    '''
+    System of losing warnings/guesses for entering letter which was guessed before
+    '''
+    if warnings_left <= 0:
+        guesses_left -= 1
+        print("Oops! You've already guessed that letter. You have no warnings left so you lose one guess:",
+              get_guessed_word(secret_word, letters_guessed))
+        return guesses_left
+    else:
+        warnings_left -= 1
+        print("Oops! You've already guessed that letter. You have", warnings_left, "warnings left:",
+              get_guessed_word(secret_word, letters_guessed))
+        return warnings_left
+
+
+def results(guessed_letter, guesses_left, letters_in_secret_word, secret_word, letters_guessed):
+    '''
+    Showing results of current guessing
+    '''
+    if guessed_letter in letters_in_secret_word:
+        # Part of code for correct guess
+        print("Good guess:", get_guessed_word(secret_word, letters_guessed))
+        return guesses_left
+    else:
+        # System of losing guesses for incorrect guessed word
+        if guessed_letter in EN_VOWELS:
+            guesses_left -= 2
+        else:
+            guesses_left -= 1
+        print("Oops! That letter is not in my word:", get_guessed_word(secret_word, letters_guessed))
+        return guesses_left
+
+
+def end_of_the_game(winning, guesses_left,secret_word):
```

Comment:
It' will look better if you add a space after comma

Reply:
done